scripts/util/create-DR.js

```diff
@@ -88,56 +88,45 @@ const createAndActivateDR = async (path, createOnly, activateOnly) => {
 
   let tx, receipt;
   // Create dispute resolver
-  if (!activateOnly) {
-    // privateKey
-    let disputeResolverSigner;
-
-    if (!privateKey) {
-      disputeResolverSigner = protocolAdminSigner;
-    } else {
-      disputeResolverSigner = new ethers.Wallet(privateKey, protocolAdminSigner.provider);
-    }
-
-    // create dispute resolver with callers account
-    let initialDisputeResolver = { ...disputeResolver };
-    initialDisputeResolver.admin = disputeResolverSigner.address;
-    initialDisputeResolver.operator = disputeResolverSigner.address;
-    initialDisputeResolver.clerk = disputeResolverSigner.address;
-
-    tx = await accountHandler
-      .connect(disputeResolverSigner)
-      .createDisputeResolver(initialDisputeResolver, disputeResolverFees, sellerAllowList);
-    receipt = await tx.wait(confirmations);
-    initialDisputeResolver = getDisputeResolverFromEvent(receipt.events, "DisputeResolverCreated", 1);
-
-    // if caller does not match supplied dispute resolver, update it.
-    // this is primary used when one does not have access to private key of dispute resolver or it does not exist (i.e. DR is a smart contract)
-    if (
-      initialDisputeResolver.admin.toLowerCase() != disputeResolver.admin.toLowerCase() ||
-      initialDisputeResolver.operator.toLowerCase() != disputeResolver.operator.toLowerCase() ||
-      initialDisputeResolver.clerk.toLowerCase() != disputeResolver.clerk.toLowerCase()
-    ) {
-      disputeResolver.id = initialDisputeResolver.id;
-      tx = await accountHandler.connect(disputeResolverSigner).updateDisputeResolver(disputeResolver);
-      receipt = await tx.wait(confirmations);
-      disputeResolver = getDisputeResolverFromEvent(receipt.events, "DisputeResolverUpdated", 1);
-    } else {
-      // no need to update on chain
-      disputeResolver = initialDisputeResolver;
-    }
+  let disputeResolverSigner;
 
-    console.log(`Dispute resolver created with id ${disputeResolver.id}`);
+  if (!privateKey) {
+    disputeResolverSigner = protocolAdminSigner;
+  } else {
+    disputeResolverSigner = new ethers.Wallet(privateKey, protocolAdminSigner.provider);
   }
```

Comment:
These lines above and the `hasRole` check are no longer needed since the DR no longer gets activated. The script errors at `hasRole`

Reply:
which network you are running? this doesn't happens to me on localhost, but I remove it anyway as isn't necessary check the role anymore